src/commands/block/tag.ts

```diff
@@ -0,0 +1,59 @@
+import { Args, Flags } from "@oclif/core";
+import { trackEvent } from "../../utils/mixpanel";
+import * as fs from "node:fs";
+import { diggerJson, updateDiggerJson } from "../../utils/helpers";
+import { BaseCommand } from "../../base";
+
+export default class Rename extends BaseCommand<typeof Rename> {
+  static description = "Add an image tag to a block";
+
+  static examples = ["<%= config.bin %> <%= command.id %>"];
+
+  static flags = {
+    tag: Flags.string({ char: "t", description: "Tag name to add." }),
+  };
+
+  static args = {
+    name: Args.string(),
+  };
+
+  public async run(): Promise<void> {
+    const { args, flags } = await this.parse(Rename);
+    trackEvent("block tag called", { flags, args });
+    if (!args.name) {
+      this.log(
+        `No app name provided. Example: dgctl block <name>`
+      );
+      return;
+    }
+
+    if (!fs.existsSync(`${process.cwd()}/${args.name}`)) {
+      this.log(`${args.name} directory not found.`);
+      return;
+    }
+
+    if (!flags.tag) {
+      this.log(
+        `No tag name provided. Example: dgctl block ${args.name} -t tag_name`
+      );
+      return;
+    }
+
+    const currentDiggerJson = diggerJson();
+    const updatedApp = currentDiggerJson.blocks.map((block: any) => {
+      if (block.name === args.name) {
+        block.ecr_image_tag = flags.tag
```

Comment:
what is the simplest way to update json file in a dir, instead of updating  'root' json ?